### DISCUSSION TITLE
Add server switching in Quick Bar, for macOS Companion app

### DISCUSSION THREAD

**Discussion by adamfeldman** [home-assistant/frontend#10967] [Other feature requests] [answered]
The macOS and iOS apps [recently gained](https://github.com/home-assistant/iOS/releases/tag/release%2F2021.12%2F2021.322) the ability to switch between servers (🎉 this is so exciting!). 

On macOS, currently that can only be accomplished from within `Preferences > Servers`; it can also be accomplished on iOS with a quick 3-finger swipe.

For macOS (and iOS), it would be great to add server switching/selection to the [Quick Bar](https://www.home-assistant.io/docs/tools/quick-bar/). 

Currently, I see nothing related to the Companion App in the Quick Bar on either iOS or macOS. 
* iOS: the Companion App settings UI is not listed in the Quick Bar menu (but is listed in the Configuration UI page)
* macOS: Preferences is not listed in the Quick Bar menu on macOS (and _is not_ listed in the Configuration UI page).

**Comment by frenck** [comment, marked as answer]
This is not a frontend request, but a request for the iOS repository and therefore I think misplaced.

https://github.com/home-assistant/iOS/discussions

**Reply by adamfeldman** [reply]
Moved this discussion, thank you! https://github.com/home-assistant/iOS/discussions/2012.

I landed here from the iOS repo, via the option for "frontend feature requests...".

![image](https://user-images.githubusercontent.com/133385/146680983-d49ed4a7-7084-44d2-b7e5-9be6afe742ff.jpeg)
